### DISCUSSION TITLE
Uploading Android library to Github Packages returns 404

### DISCUSSION THREAD

**Discussion by iadcialim** [community/community#26290] [Actions]
<p>Hi. Im new to Github Packages and to this community.<br>
I was trying to upload my android library from a private repo which belongs to my company’s Github account.<br>
I am also following this tutorial:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://proandroiddev.com/publishing-android-libraries-to-the-github-package-registry-part-1-7997be54ea5a">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/14870470/181096693-c9323e14-7144-4871-9594-93c2c4cd0a04.png" class="site-icon" width="256" height="256">

      <a href="https://proandroiddev.com/publishing-android-libraries-to-the-github-package-registry-part-1-7997be54ea5a" target="_blank" rel="noopener nofollow ugc" title="08:55PM - 11 October 2020">Medium – 11 Oct 20</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/459;"><img src="https://user-images.githubusercontent.com/14870470/181096694-70365824-6307-4555-8f76-e7a4aa06c149.jpeg" class="thumbnail" width="690" height="459" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/1/f1732faae9b4d287d1f8d60459d9e61290c95eb4_2_690x459.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/f/1/f1732faae9b4d287d1f8d60459d9e61290c95eb4_2_1035x688.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/1/f1732faae9b4d287d1f8d60459d9e61290c95eb4.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/1/f1732faae9b4d287d1f8d60459d9e61290c95eb4_2_10x10.png"></div>

<h3><a href="https://proandroiddev.com/publishing-android-libraries-to-the-github-package-registry-part-1-7997be54ea5a" target="_blank" rel="noopener nofollow ugc">Publishing Android libraries to GitHub Packages</a></h3>

  <p>Github has recently introduced the GitHub Packages, a package management service where developers and organisations can publish…</p>

  <p>
    <span class="label1">Reading time: 4 min read</span>
  </p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Error I got:</p>
<pre><code class="lang-auto">Execution failed for task ':library:publishBarPublicationToGitHubPackagesRepository'.
&gt; Failed to publish publication 'bar' to repository 'GitHubPackages'

&gt; Could not PUT 'https://maven.pkg.github.com/ORG/REPO_PATH.git/AA/BB/CC/ARTIFACT_ID/0.0.1/LIB.aar'. Received status code 404 from server: Not Found
</code></pre>
<p>where groupId is “AA.BB.CC”</p>
<p>Is there any other step to do in Github?</p>

**Comment by iadcialim** [comment, marked as answer]
<p>Never mind, I have solved it. Just need to remove the <code>.git</code> part</p>

**Comment by tuves** [comment]
<p>Thank you for circling back with your solution, <a class="mention" href="/iadcialim">@iadcialim</a>! :slight_smile:</p>
